javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,146 @@
+/*freecodecamp.org account name: mehmetnuricavdar
+https://www.freecodecamp.org/mehmetnuricavdar
+*/
+
+// ######    Flight booking fullname function
+
+function getFullname(firstname, surname) {
+  const fullName = `${firstname} ${surname}`;
+  return fullName;
+}
+
+let fullname1 = getFullname("Benjamin", "Hughes");
+let fullname2 = getFullname("Mehmet", "Cavdar");
+
+// ###### Formal fullname
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  const fullName = `${firstname} ${surname}`;
```

Comment:
Can't you reuse your previous function here to get the fullname ?

Reply:
i just did step by step so did full function after that but i deleted it since no need for that